kingpin/actors/rightscale/base.py

```diff
@@ -195,3 +197,18 @@ def flatten(d, parent_key=prefix, sep='_'):
             return items
 
         return flatten(params)
+
+    @gen.coroutine
+    def _log_account_name(self, *args, **kwargs):
+        if not RightScaleBaseActor.account_name:
+            ca_resource = self._client._client.cloud_accounts
+            cloud_accounts = yield self._client.show(ca_resource)
+            account = yield self._client.show(cloud_accounts[0].account)
+            RightScaleBaseActor.account_name = account.soul['name']
+            log.warn('RightScale account name: %s' % account.soul['name'])
+
+    @gen.coroutine
+    def execute(self, *args, **kwargs):
+        self._log_account_name()
```

Comment:
@diranged Not happy with this :( Would appreciate some feedback here.

I wanted the account name to be printed exactly once, on the first use of *any* rightscale actor. This implementation does it.

But then I would have to mock this out in **every single test** :( So what I did is remove the yield here. 

Can't have it in __init__ because it would be synchronous and blocking and I would still have to mock it in every test.